packages/min-100-layout/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import s from './style.module.css'
+
+interface Min100LayoutProps {
+  children: React.ReactNode
+  footer: React.ReactNode
+}
+
+function Min100Layout({
+  children,
+  footer,
+}: Min100LayoutProps): React.ReactElement {
+  return (
+    <div className={s.root}>
+      <div>{children}</div>
```

Comment:
Wrapped `children` here because rendering directly into a `flex` container felt like it might be a bit unexpected.